d2core/d2audio/sound_engine.go

```diff
@@ -32,6 +37,8 @@ type Sound struct {
 	vRate   float64
 	state   envState
 	// panning float64 // lets forget about this for now
+
+	logger *d2util.Logger
 }
```

Comment:
same as audio provider, can just embed with the name to make usage less verbose.